### ISSUE TITLE
[Windows] The tooltips regarding command overrides are not read verbally on screen readers

### ISSUE THREAD

**Issue by Gravecat** [Gravecat/Greave#26]
What are the steps to reproduce this issue?
-------------------------------------------
1. Enable screen reader support (create a `userdata/prefs.yml` file containing `screen_reader_sapi: true`
2. Get into a fight with any NPC
3. Attempt to leave

What happens?
-------------
While attempting to leave, the enemy's attack will interrupt your action, resulting in a message telling the player they can put ! in front of their command to force it. This is in `Parser::interrupted()`. A similar message is also seen in `Parser::confirm_message()`.

What were you expecting to happen?
----------------------------------
Ideally the line should be read out loud, but on screen readers, the `!` part is silent.

Any logs, error output, etc?
----------------------------
![greave3](https://user-images.githubusercontent.com/1296718/125363829-41351080-e369-11eb-8eb2-9224c19db485.png)

Any other comments?
-------------------
Ideally, the two Parser functions should be edited so that if a screen reader is in use, it'll explicitly state that an exclamation mark should be added, since it doesn't seem to want to do so automatically.

What versions of software are you using?
----------------------------------------
**Operating System:** Windows 10 64-bit

**Toolchain Version:** MinGW-gcc 8.1.0, CMake 3.18.3

**Comment by Gravecat** [comment]
Okay, this one turned out to be a surprisingly easy fix.
